docs/sources/reference/commandline/cli.md

```diff
@@ -17,6 +17,30 @@ or execute `docker help`:
 
       ...
 
+## Environment Variables
+
+For easy reference, the follow list of environment variables will be used by
```

Comment:
Typo: following
I recommend rephrasing for conditional since all of these won't always be used: "...variables can be used by the `docker`..." Or maybe, "...are supported by..."?